CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(DEVELOPMENT_BUILD TRUE)
 
 set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 if(VERSION_EXTRA)
-	set(VERSION_STRING ${VERSION_STRING}-${VERSION_EXTRA})
+	set(VERSION_STRING "${VERSION_STRING}-${VERSION_EXTRA}")
```

Comment:
Please explain in the commit message what bug this change fixes. It is not obvious to me.

Reply:
I don't think there's an actual bug here but it's better to quote the arguments.